Source/NSSplitViewItem.m

```diff
@@ -0,0 +1,177 @@
+/* Implementation of class NSSplitViewItem
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: Mon 20 Jul 2020 12:56:20 AM EDT
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#import <Foundation/NSArchiver.h>
+#import "AppKit/NSSplitViewItem.h"
+
+@implementation NSSplitViewItem
+- (instancetype) initWithViewController: (NSViewController *)viewController
+{
+  self = [super init];
+  if (self != nil)
+    {
+      ASSIGN(_viewController, viewController);
+    }
+  return self;
+}
+
++ (instancetype) contentListWithViewController: (NSViewController *)viewController
+{
+  return [[NSSplitViewItem alloc] initWithViewController: viewController];
```

Comment:
In the following methods we need to set a few more attributes for the created objects.

Reply:
And of course the returned values should be autoreleased.

Reply:
Done.